.github/labeler.yml

```diff
@@ -0,0 +1,35 @@
+go:
+- '**/*.go'
+- 'go.mod'
+- 'go.sum'
+
+proto:
+- '**/*.proto'
+- 'buf.*'
+
+javascript:
+- '**/*.js'
+- '**/*.ts'
+- '**/package.json'
+- '**/package-lock.json'
+
+solidity:
+- '**/*.sol'
+
+dependencies:
+- 'go.mod'
+- 'go.sum'
+- '**/package.json'
+- '**/package-lock.json'
+
+docker:
+- '**/Dockerfile'
+- '**/docker-compose.yml'
+
+actions:
+- '.github/actions/**'
+- '.github/workflows/**'
+
+repo:
+- '*'
```

Comment:
Is this not gonna tag all the PRs with the `repo` tag?

Reply:
Should just be impacting files in the root but no others, which would in theory be specified by the globstar `**/*`. Now that I'm looking at it a second time though I might make it not track `go.mod`-esque files. One sec.